cmd/zfs/zfs_main.c

```diff
@@ -4328,6 +4328,15 @@ zfs_do_send(int argc, char **argv)
 		}
 	}
 
+	if (flags.dedup) {
+		(void) fprintf(stderr,
+		    gettext("WARNING: deduplicated send is "
+		    "deprecated, and will be removed in a\n"
+		    "future release. (The flag will be accepted, but "
+		    "a regular,\n"
+		    "non-deduplicated stream will be generated.)\n\n"));
```

Comment:
This language is slightly confusing to me, in that it implies that we're currently generating a non-deduplicated stream. Just prepending "In the future" on the parenthetical would make it a bit clearer imo. Feel free to ignore, though.

Reply:
Sure thing.